# Test cases automatically generated by Pynguin (https://www.pynguin.eu).
# Please check them before you use them.
import pytest
import example as module_0


@pytest.mark.xfail(strict=True)
def test_case_0():
    bool_0 = True
    module_0.meetup(bool_0, bool_0, bool_0, bool_0)


def test_case_1():
    int_0 = -1633
    meetup_day_exception_0 = module_0.MeetupDayException(int_0)
    assert meetup_day_exception_0.message == -1633


@pytest.mark.xfail(strict=True)
def test_case_2():
    int_0 = -1633
    module_0.meetup(int_0, int_0, int_0, int_0)


@pytest.mark.xfail(strict=True)
def test_case_3():
    str_0 = "last"
    bool_0 = True
    module_0.meetup(bool_0, bool_0, str_0, str_0)


@pytest.mark.xfail(strict=True)
def test_case_4():
    str_0 = "Thursday"
    none_type_0 = None
    meetup_day_exception_0 = module_0.MeetupDayException(none_type_0)
    assert meetup_day_exception_0.message is None
    meetup_day_exception_1 = module_0.MeetupDayException(str_0)
    assert meetup_day_exception_1.message == "Thursday"
    bool_0 = True
    module_0.meetup(bool_0, bool_0, str_0, str_0)


@pytest.mark.xfail(strict=True)
def test_case_5():
    str_0 = "teenth"
    bool_0 = True
    module_0.meetup(bool_0, bool_0, str_0, bool_0)
